52/abkosar/timer.py

```diff
@@ -0,0 +1,30 @@
+import argparse
+import time
+import os
+from datetime import datetime, timedelta
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description='Pomodoro Timer')
```

Comment:
You might consider refactoring your argument parsing into its own function.

Reply:
Great advice, definitely will do that!